### PR TITLE
Add TEE Python Hello World example

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 1]
```diff
@@ -115,7 +134,52 @@ docker run -it --rm \
...
+IMG_NAME=python-hello-world
+
+IMG_FROM=${IMG_NAME}:temp-non-tee
+IMG_TO=${IMG_NAME}:tee-debug
+
+# build the regular non-TEE image
+docker build . -t ${IMG_FROM}
+
+# pull the SCONE curated image corresponding to our base image
+docker pull registry.scontain.com:5050/sconecuratedimages/apps:python-3.7.3-alpine3.10-scone5
```
Comment (zguesmi) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]: Is this required for Python apps?

Comment (mcornaton) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: I thought so but it looks like I was wrong. What's the point of this line in the Node version?

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 2]
```diff
@@ -59,6 +59,23 @@ ENTRYPOINT [ "node", "/app/app.js"]
...
 {% endtab %}
+
+{% tab title="Python" %}
+{% code title="Dockerfile" %}
+```bash
+# Starting from a base image supported by SCONE  
+FROM python:3.7-alpine3.10
+
+# install your dependencies
+RUN apk add gcc musl-dev
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]: I think we don't need it (https://github.com/iExecBlockchainComputing/python-hello-world/blob/main/cloud-computing/Dockerfile)

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 3]
```diff
@@ -59,6 +59,23 @@ ENTRYPOINT [ "node", "/app/app.js"]
...
+RUN pip3 install eth_abi
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]: pyfiglet dependency is missing (should match https://docs.iex.ec/for-developers/your-first-app#dockerize-your-app & https://github.com/iExecBlockchainComputing/python-hello-world/blob/main/cloud-computing/Dockerfile)

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 2 above)
Comment (mcornaton) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: You're right. That's an oversight; I didn't copy/paste the right example.

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 3 above)
Comment (mcornaton) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: Same here. That's not the right one.

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 2 above)
Comment (zguesmi) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: To be confirmed, but I think these dependencies are needed in order for `eth_abi` to work.

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 1 above)
Comment (zguesmi) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: It is a Scone detail. I think it is only required for Nodejs apps.

**Review by james-toussaint — APPROVED** [review]
Thanks!

**Pull request merged**